product/stream_processor/domain_logic/product_notification.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from product.models.products.product import ProductChangeNotification
+from product.stream_processor.integrations.events.event_handler import ProductChangeNotificationHandler
+from product.stream_processor.integrations.events.models.output import EventReceipt
+from product.stream_processor.integrations.events.providers.eventbridge import EventBridge
+
+EVENT_BUS = os.environ.get('EVENT_BUS', '')
+EVENT_SOURCE = 'myorg.product.product_notification'
+
+
+def notify_product_updates(update: list[ProductChangeNotification], event_handler: ProductChangeNotificationHandler | None = None) -> EventReceipt:
+    """Notify product change notifications using default or provided event handler.
+
+    Parameters
+    ----------
+    update : list[ProductChangeNotification]
```

Comment:
general question: why do you write list[X] onstead of List[X], same for dict, tuple etc. i've ppl do it but dunno if it matters

Reply:
because it was deprecated since 3.9 :) We use `typing.List` for libraries or applications that need to support 3.7-3.8. We're 3.11 so we get to Live On THe Edge!!!!

https://docs.python.org/3.10/library/typing.html#typing.List